pmd-html/src/test/java/net/sourceforge/pmd/lang/html/HtmlXPathRuleTest.java

```diff
@@ -24,25 +20,42 @@ public class HtmlXPathRuleTest {
     private static final String LIGHTNING_WEB_COMPONENT = "<!-- helloExpressions.html -->\n"
             + "<template>\n"
             + "    <p>Hello, { greeting}!</p>\n"
-            + "    <lightning-input label=\"Name\" value={ greeting} onchange={handleChange}></lightning-input>"
+            + "    <lightning-input label=\"Name\" value={ greeting} onchange={handleChange}></lightning-input>\n"
             + "    <div class=\"slds-m-around_medium\">\n"
-            + "        <lightning-input name='firstName' label=\"First Name\" onchange={handleChange}></lightning-input>\n"
+            + "        <lightning-input name='firstName' label=\"First Name\" onchange={ handleChange}></lightning-input>\n"
             + "        <lightning-input name='lastName' label=\"Last Name\" onchange={handleChange}></lightning-input>\n"
             + "        <p class=\"slds-m-top_medium\">Uppercased Full Name: {uppercasedFullName}</p>\n"
             + "    </div>\n"
+            + "    <template if:true={visible}>\n"
+            + "      <p>Test</p>\n"
+            + "    </template>\n"
             + "</template>";
 
     @Test
     public void selectTextNode() {
         // from https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.js_props_getter
         // "Don’t add spaces around the property, for example, { data } is not valid HTML."
-        String xpath = "//*[local-name() = '#text'][contains(@Text, '{ ')]";
+        String xpath = "//text()[contains(., '{ ')]";
```

Comment:
cool :) Can we also assert that the previous query doesn't match anything? Ie, saxon does not see elements with name `#text` or `#comment`

Reply:
I've added another test case - it works, `#text` doesn't match anything anymore.